tests/reference/run-array_shape_01_cc-098a6d5.json

```
@@ -0,0 +1,13 @@
+{
+    "basename": "run-array_shape_01_cc-098a6d5",
+    "cmd": "lfortran --continue-compilation --no-color {infile}",
+    "infile": "tests/errors/array_shape_01_cc.f90",
+    "infile_hash": "06b15e2c80ebbe32eefcfcf9b604aba89526d7bb6aa285dbf99b6b27",
+    "outfile": null,
+    "outfile_hash": null,
+    "stdout": null,
+    "stdout_hash": null,
+    "stderr": "run-array_shape_01_cc-098a6d5.stderr",
+    "stderr_hash": "3e79bec3aecb83e04c123d1b1759f7600f44c99bc1eef1627977bde9",
+    "returncode": 134
```

```suggestion
    "returncode": 134
```
It should have been 1. Checking what went wrong.

---

That was due to wrong initialization of x. Corrected that.